cht/adapter.comm

```diff
@@ -134,11 +134,11 @@ while adapter.isCouplingOngoing():
         # Output if necessary
         k += 1
         if k % settings["simulation"]["output-frequency"] == 0:
-               filename = "output-" + str(k) + ".rmed"
-               DEFI_FICHIER(FICHIER=filename, UNITE=80)
-               IMPR_RESU(FORMAT='MED', RESU=_F(RESULTAT=TEMP))
-               call(["mv", filename, settings["base-path"] + "/" + participant["directory"]])
-               DEFI_FICHIER(ACTION='LIBERER', UNITE=80)
+               filename = "REPE_OUT/output-" + str(k) + ".rmed"
+               DEFI_FICHIER(FICHIER=filename, UNITE=80+k, TYPE='BINARY')
+               IMPR_RESU(FORMAT='MED', UNITE=80+k, RESU=_F(RESULTAT=TEMP))
+               #call(["mv", filename, settings["base-path"] + "/" + participant["directory"]])
```

Comment:
Why is this commented out? What does it do? Why not remove it completely?

Reply:
Removed it completely